blackbox/docs/configuration.txt

```diff
@@ -493,6 +493,18 @@ Repositories are used to :ref:`backup <snapshot-restore>` a CrateDB cluster.
 
 See also the :ref:`path.repo <conf-path-repo>` Setting.
 
+Queries
+-------
+
+.. _conf-indices-query-bool.max_clause_count:
+
+**indices.query.bool.max_clause_count**
```

Comment:
We need to implement this setting "from scratch", but as discussed we'll stick to the current behaviour for the time being.